src/main/java/com/jcabi/github/Commits.java

```diff
@@ -39,7 +39,7 @@
  *
  * @author Carlos Miranda (miranda.cma@gmail.com)
  * @version $Id$
- * @since 0.8
+ * @since 0.23
```

Comment:
@zygm0nt we shouldn't change this line, because it tells our user when this _class_ was added to the library. Instead, as @longtimeago suggested, we should add a new `@since` line to the javadoc of the method you're introducing